priam/src/main/java/com/netflix/priam/restore/AbstractRestore.java

```diff
@@ -245,9 +245,11 @@ public void restore(DateUtil.DateRange dateRange) throws Exception {
             List<AbstractBackupPath> allFiles =
                     BackupRestoreUtil.getMostRecentSnapshotPaths(
                             latestValidMetaFile.get(), metaProxy, pathProvider);
-            allFiles.addAll(
-                    BackupRestoreUtil.getIncrementalPaths(
-                            latestValidMetaFile.get(), dateRange, metaProxy));
+            if (!config.skipIncrementalRestore()) {
+                allFiles.addAll(
+                        BackupRestoreUtil.getIncrementalPaths(
+                                latestValidMetaFile.get(), dateRange, metaProxy));
+            }

```

Comment:
I take it we don't have to update the callsite in BackupServletV2?

Reply:
I updated it there as well.

Reply:
I'm confused - do we need to use `config.skipIncrementalRestore` there too? I didn't see that there in this PR, perhaps it isn't required?